huaweicloud/services/acceptance/dds/data_source_huaweicloud_dds_flavors_v3_test.go

```diff
@@ -0,0 +1,35 @@
+package dds
+
+import (
+	"testing"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDDSFlavorV3DataSource_basic(t *testing.T) {
+	dataSourceName := "data.huaweicloud_dds_flavors.flavor"
+	dc := acceptance.InitDataSourceCheck(dataSourceName)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDDSFlavorV3DataSource_basic,
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(
+						dataSourceName, "engine_name", "DDS-Community"),
```

Comment:
this is not a valid check, it is recommended to add a check for `flavors`.

Reply:
fixed